build.sh

```diff
@@ -14,19 +14,10 @@ while [ $# -gt 0 ]; do
   shift
 done
 
-cd legacy
-
-bundle install
-bundle exec jekyll build
-
-cd ../
-
 if [ "$mode" == "development" ]; then
   yarn build --mode "$mode" --site "http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com"
```

Comment:
@AndyClifford Suggestion: Set `--mode` and `--site` in the Github actions and delete this script

Reply:
And copy the CNAME in Github workflow too?